css/global.css

```
@@ -0,0 +1,57 @@
+:root {
+  --primary-100: #3692ff;
+  --primary-200: #1967d6;
+  --primary-300: #1251aa;
+  --main-bg-color: #cfe5ff;
+  --secondary-gray-900: #111827;
+  --secondary-gray-800: #1f2937;
+  --secondary-gray-700: #374151;
+  --secondary-gray-600: #4b5563;
+  --secondary-gray-500: #6b7280;
+  --secondary-gray-400: #9ca3af;
+  --secondary-gray-200: #e5e7eb;
+  --secondary-gray-100: #f3f4f6;
+  --secondary-gray-50: #f9fafb;
+  --error-red: #f74747;
+  --main-font: Pretendard;
+  --size-max-width: 1200px;
+}
```

P3:
한가지 폰트만 사용되니 변수로 폰트명을 선언해두시고 사용하기보다 그냥 바로 스타일링해주셔도 괜찮을 것 같습니다.

```suggestion
  --size-max-width: 1200px;
}

html {
  font-family: "Pretendard", sans-serif
}
```